src/realm/sync/noinst/client_impl_base.cpp

```diff
@@ -731,6 +731,11 @@ void Connection::handle_connection_established()
 
     m_state = ConnectionState::connected;
 
+    // TODO(RCORE-1380) get this information in-band rather than from the websocket.
+    if (auto coid = m_websocket->get_appservices_request_id(); !coid.empty()) {
+        logger.info("Connected to app services with request id: \"%1\"", coid);
```

Comment:
Do you want to print something if `coid` is empty instead of nothing at all?

Reply:
I think that would end up being kinda noisy.